lib/datadog/tracing/contrib/pg/instrumentation.rb

```diff
@@ -81,10 +81,12 @@ def sync_exec_prepared(statement_name, params, *args, &block)
 
             def trace(name, sql: nil, statement_name: nil, block: nil)
               service = Datadog.configuration_for(self, :service_name) || datadog_configuration[:service_name]
+              error_handler = datadog_configuration[:error_handler]
```

Comment:
Please let me know if there is another standard way to get the error handler here.